lifelines/_univariate_weibull.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
```

Comment:
Should add

    from __future__ import division

to avoid differences with python3.